entproto/cmd/protoc-gen-entgrpc/template/method_batch_create.tmpl

```diff
@@ -0,0 +1,74 @@
+{{- /*gotype: entgo.io/contrib/entproto/cmd/protoc-gen-entgrpc.methodInput*/ -}}
+{{ define "method_batch_create" }}
+
+    {{- $idField := .G.FieldMap.ID -}}
+    {{- $varName := $idField.EntField.Name -}}
+    {{- $inputName := .Method.Input.GoIdent.GoName -}}
+    {{- $methodName := .Method.GoName -}}
+    {{- $reqVar := camel .G.EntType.Name -}}
+    requests := req.GetRequests()
+    if len(requests) > entproto.MaxBatchCreateSize {
+        return nil, {{ statusErrf "InvalidArgument" "batch size cannot be greater than %d" "entproto.MaxBatchCreateSize" }}
+    }
+    bulk := make([]*ent.{{ .G.EntType.Name }}Create, len(requests))
+    for i, req := range requests {
+        bulk[i] = svc.client.{{ .G.EntType.Name }}.Create()
+        m := bulk[i]
+        {{ $reqVar }} := req.Get{{ .G.EntType.Name }}()
```

Comment:
This section is copied directly from the mutate template. They are different enough that it didn't make sense to put the batch create method into method_mutate.tmpl. However, we could refactor to move this section into its own template that can be called by method_batch_create.tmpl and method_mutate.tmpl.

Reply:
Yes, feels wrong to duplicate the code both in the templates and then again in the generated code. 

I'm in favor of extracting this part into a private helper function that will be generated if either Create or BatchCreate are wanted by the user and used by both.